### PR TITLE
added packages/dokeysto/dokeysto.3.0.2

### PR DESCRIPTION
**Pull request by UnixJunkie**
no more dependency on dolog

### CODE REVIEW

File: packages/dokeysto/dokeysto.3.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+bug-reports: "https://github.com/UnixJunkie/dokeysto/issues"
+dev-repo: "git+https://github.com/UnixJunkie/dokeysto.git"
+license: "LGPL-2.1"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test"] {with-test}
+]
+depends: [
+  "ocaml"
+  "dune"
```
Comment (kit-ty-kate) [inline packages/dokeysto/dokeysto.3.0.2/opam]:
```suggestion
  "dune" {>= "1.11"}
```
The `dune-project` file in your repository says this package requires at least dune 1.11

File: packages/dokeysto/dokeysto.3.0.2/opam [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+name: "dokeysto"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/dokeysto"
+bug-reports: "https://github.com/UnixJunkie/dokeysto/issues"
+dev-repo: "git+https://github.com/UnixJunkie/dokeysto.git"
+license: "LGPL-2.1"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test"] {with-test}
```
Comment (kit-ty-kate) [inline packages/dokeysto/dokeysto.3.0.2/opam]:
```suggestion
  ["dune" "exec" "-p" name "-j" jobs "src/test.exe"] {with-test}
```
`dune build` alone without `-p` or `--profile=release` (as used in `make test`) is not meant for releases and might to break things in the future.

Comment (kit-ty-kate) [reply packages/dokeysto/dokeysto.3.0.2/opam]: Also i would suggest to use the test feature from dune next time: https://dune.readthedocs.io/en/stable/quick-start.html#defining-tests

**Comment by UnixJunkie** [comment]
@kit-ty-kate thanks for the fixes in the opam file. They were all included.

**Comment by kit-ty-kate** [comment]
Thanks a lot!

**Pull request merged**